.github/workflows/build.yml

```diff
@@ -12,7 +12,6 @@ jobs:
         with:
           repository: mideind/Yfirlestur
           path: yfirlestur
-          ref: "dockerfile_development"
```

Comment:
best to use a defined SHA1, so your tests don't break if sth. fundamental changes on master

Reply:
Will address this in the test PR as it is more appropriate there